src/main/java/frc/robot/Robot.java

```diff
@@ -74,8 +58,6 @@ public void robotInit() {
     // Configure the initial Pose (field position, angle) of the robot
     // For now, we're hardcoding the pose. Eventually we will use a selected robot pose
     // m_robotContainer.robotDrive.setPose(FieldMap.startPositions[1]);
```

Comment:
Maybe delete this?

Reply:
Good catch. I removed that, and also fixed the comment above robotPeriodic().